region/maridia/inner-yellow.json

```diff
@@ -263,12 +263,12 @@
                 {
                   "name": "Bug Room Charge Out Left with Hijump",
                   "notable": false,
-                  "note": "Charge below then climb up and carry the spark out of the left door.",
                   "requires": [
                     "canShinechargeMovementComplex",
                     "HiJump",
                     "canWalljump"
-                  ]
+                  ],
+                  "note": "Charge below then climb up and carry the spark out of the left door."
```

Comment:
Charge a spark left and use the last bit of platform to jump up to the ledge above